senders/formatter.go

```diff
@@ -125,18 +125,18 @@ func HistoLine(name string, centroids histogram.Centroids, hgs map[histogram.Gra
 //    parent=2f64e538-9457-11e8-9eb6-529269fb1459 application=Wavefront http.method=GET 1533531013 343500"
 func SpanLine(name string, startMillis, durationMillis int64, source, traceId, spanId string, parents, followsFrom []string, tags []SpanTag, spanLogs []SpanLog, defaultSource string) (string, error) {
 	if name == "" {
-		return "", errors.New("empty span name")
+		return "", errors.New("span name cannot be empty")
 	}
 
 	if source == "" {
 		source = defaultSource
 	}
 
 	if !isUUIDFormat(traceId) {
-		return "", errors.New("traceId is not in UUID format")
+		return "", fmt.Errorf("traceId is not in UUID format: span=%s", name)
```

Comment:
Would it be worth showing the trace ID in the error?

Reply:
Good idea – I added it.